networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/libp2p/LibP2PNetwork.java

```diff
@@ -149,11 +151,11 @@ public LibP2PNetwork(
     if (!state.compareAndSet(State.IDLE, State.RUNNING)) {
       return SafeFuture.failedFuture(new IllegalStateException("Network already started"));
     }
-    STATUS_LOG.log(Level.INFO, "Starting libp2p network...");
+    LOG.info("Starting libp2p network...");
     return SafeFuture.of(host.start())
         .thenApply(
             i -> {
-              STATUS_LOG.log(Level.INFO, "Listening for connections on: " + getNodeAddress());
+              LOG.info("Listening for connections on: {}", getNodeAddress());
```

Comment:
This is a really, really useful thing to have printed to stdout.

Reply:
Actioned